### PR TITLE
Import ConfigureUtil explicitly

### PR DESCRIPTION
**Pull request by sabify**
Gradle can't resolve ConfigureUtil implicitly.
Error:
Script 'firebase_cpp_sdk/Android/firebase_dependencies.gradle' line: 84
* What went wrong:
A problem occurred evaluating project ':Sample'.
> Could not get unknown property 'ConfigureUtil' for object of type FirebaseCppExtension.

### CODE REVIEW

**Comment by sabify** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here (e.g. `I signed it!`) and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google.com/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ffirebase%2Ffirebase-cpp-sdk%2Fpull%2F12) for more info**.

I signed it!

File: Android/firebase_dependencies.gradle [hunk 1]
```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 // A map of library to the dependencies that need to be added for it.
+import org.gradle.util.*;
```
Comment (crazyhappygame) [inline Android/firebase_dependencies.gradle]: consider change to
`import org.gradle.util.ConfigureUtil;`

Comment (sabify) [reply Android/firebase_dependencies.gradle]: updated 😉

**Comment by crazyhappygame** [comment]
@sabify It looks like this PR can be closed. Below line was added to latest version
`import org.gradle.util.ConfigureUtil;`

**Pull request closed without merging**